app/src/main/java/org/jellyfin/androidtv/browsing/HomeFragment.java

```diff
@@ -365,20 +363,9 @@ protected void addAdditionalRows(ArrayObjectAdapter rowAdapter) {
         toolsRow = new ArrayObjectAdapter(mGridPresenter);
         toolsRow.add(new GridButton(SETTINGS, mApplication.getString(R.string.lbl_app_settings), R.drawable.gears));
         toolsRow.add(new GridButton(LOGOUT, mApplication.getString(R.string.lbl_logout) + TvApp.getApplication().getCurrentUser().getName(), R.drawable.logout));
-
-        sendLogsButton = new GridButton(REPORT, mApplication.getString(R.string.lbl_send_logs), R.drawable.upload);
```

Comment:
Can we remove the `sendLogsButton` variable entirely?

Reply:
Derp. I've now removed the instance variable for sendLogsButton, and also the "REPORT" constant. Android Studio also suggested converting toolsRow to a local variable, so I made that change, too.